release-tools/ci/conclude.bash

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash -x
+
+set -euo pipefail
+
+VERSION=$1
+echo "You want me to conclude ${VERSION} ??"
+
+export MAVEN_HOME="$HOME/.sdkman/candidates/maven/current"
+export JAVA_HOME="$HOME/.sdkman/candidates/java/current"
+export PATH="$MAVEN_HOME/bin:$JAVA_HOME/bin:$PATH"
+
+export JENKINS_HOME=/tmp/jenkins-home
+export RELEASE_TOOLS_CACHE=${JENKINS_HOME}/.m2/spring-data-release-tools
+export LOGS_DIR=${JENKINS_HOME}/spring-data-shell/logs
+export SETTINGS_XML=${JENKINS_HOME}/settings.xml
+export GNUPGHOME=/tmp/gpghome
+
+if test -f application-local.properties; then
+    echo "You are running from dev environment! Using application-local.properties."
+
+    GIT_BRANCH=""
+
+    function spring-shell {
+        java \
+            "-Ddeployment.settings-xml=${SETTINGS_XML}" \
+            "-Ddeployment.local=true" \
+            "-Dmaven.mavenHome=${MAVEN_HOME}" \
+            "-Dgpg.executable=/usr/bin/gpg" \
+            "-Dio.workDir=dist" \
+            -jar target/spring-data-release-cli.jar
+    }
+else
+    echo "You are running inside Jenkins! Using parameters fed from the agent."
+
+    function spring-shell {
+        java \
+            "-Dgit.username=${GIT_USERNAME}" \
```

Comment:
Can we please put these placeholders into a `application-jenkins.properties` file to avoid repetition? We would then run the application with a `jenkins` profile.